### PR TITLE
Add query-chatgpt extension

### PR DESCRIPTION
**Pull request by cyxn**
## Description

Open ChatGPT in the browser from raycast with your custom prompt and specific query.
The extension leverages AppleScript to open a URL and execute JavaScript within a browser tab, automating the process of
inserting your prompt and query, and then pressing the submit button.
You can create multiple custom commands to open specific GPT versions or to open existing chats, using your
custom prompt and an additional query that you provide on the fly while running the command.

### Motivation
ChatGPT doesn't have ability to provide custom search query to the URL, like for perplexity or other search
engines `https://www.perplexity.ai/search?q=myquery`.

This extension enables the ability to query ChatGPT asynchronously using your ChatGPT subscription, eliminating the need
to wait for a response.

For example, when writing code, if I identify a utility function that can be entirely delegated to ChatGPT,
I use this extension to dispatch the command and don't wait for the response. I continue writing other parts
of the code, while a freshly opened tab in the background generates a response for me. During this, I may identify other
parts that require consultation with ChatGPT or need code to be written, and with this extension, you can open even more
tabs with ChatGPT. My workflow doesn't rely heavily on immediate responses; in fact, GPT-4 often takes too long to
respond (5+ seconds) for me to wait.

## Screencast

https://share.cleanshot.com/4bQ2Kl3r

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by pernielsentikaer** [comment]
Hi 👋

I couldn't get this to work, could you make me a screencast of the two commands?

**Comment by cyxn** [comment]
Hi @pernielsentikaer .
Did you see https://share.cleanshot.com/4bQ2Kl3r that completely demonstrates the flow?

File: extensions/query-chatgpt/package.json [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+      "arguments": [
+        {
+          "name": "prompt",
+          "placeholder": "Prompt",
+          "type": "text",
+          "required": true
+        },
+        {
+          "name": "gptUrl",
+          "placeholder": "GPT Url",
```
Comment (pernielsentikaer) [inline extensions/query-chatgpt/package.json:38]:
```suggestion
          "placeholder": "GPT URL",
```

File: extensions/query-chatgpt/package.json [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+          "type": "text",
+          "required": true
+        },
+        {
+          "name": "query",
+          "placeholder": "Your query",
```
Comment (pernielsentikaer) [inline extensions/query-chatgpt/package.json:44]:
```suggestion
          "placeholder": "Your Query",
```

File: extensions/query-chatgpt/src/query-chatgpt-custom-create.tsx [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+  }
+
+  return (
+    <Form
+      isLoading={false}
+      actions={<CustomCreateAction prompt={prompt} gptUrl={gptUrl} withCustomQuery={withCustomQuery} />}
+    >
+      <Form.TextArea
+        id="gptUrl"
+        title="GPT Url"
```
Comment (pernielsentikaer) [inline extensions/query-chatgpt/src/query-chatgpt-custom-create.tsx:51]:
```suggestion
        title="GPT URL"
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 I saw the video and it makes sense, only a few NITs and we're ready to go

I have now tested your extension, and I have some feedback ready for you 🙂

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**